pkg/readiness/ready_tracker.go

```diff
@@ -689,58 +732,82 @@ func (t *Tracker) trackConstraintTemplates(ctx context.Context) error {
 	return nil
 }
 
-// trackConfig sets expectations for cached data as specified by the singleton Config resource.
-// Fails-open if the Config resource cannot be fetched or does not exist.
-func (t *Tracker) trackConfig(ctx context.Context) error {
-	var wg sync.WaitGroup
+// trackSyncSources sets expectations for cached data as specified by the singleton Config resource.
```

Comment:
It's nit as we refer to syncset objects as syncSources already so might be good to be explicitly here just to be clear this is for both config and syncset

```suggestion
// trackSyncSetAndConfig sets expectations for cached data as specified by the singleton Config resource.
```

Reply:
This and the other suggestion I did locally as the names were linked in other places too.